packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -47,6 +50,7 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
     private readonly viewportResizeObserver: ResizeObserver;
     private virtualizer?: TanStackVirtualizer<HTMLElement, HTMLElement>;
     private _pageSize = 0;
+    private isScrollingTimeout = 0;
```

Comment:
nit: to me this name makes it seem like this is a duration but the value returned from setTimeout is actually an integer that uniquely identifies the timer. Maybe `isScrollingTimer` or `isScrollingTimeoutId`? MDN is a bit inconsistent but they sometimes use "timeout id" (and sometimes use timeout as you are, so maybe this is just my hangup).

Reply:
Agreed. Went with ` isScrollingTimer`.